### PR TITLE
increment/decrement dislike count in UI when clicked

### PR DESCRIPTION
**Pull request by priyavrat-misra**

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
                 "content.js"
             ],
             "matches": [
-                "https://leetcode.com/problems/*/description/*"
+                "https://leetcode.com/problems/*"
```
Comment (priyavrat-misra) [inline manifest.json:21]: changed because the count wasn't showing up when a question was opened without `/description/`.

File: content.js [hunk 2]
```diff
@@ -44,22 +45,32 @@ function waitForElm(selector) {
...
     console.log('Page reloaded');
     let selector = document.querySelector("[data-icon='thumbs-down']");
-    selector.insertAdjacentHTML('afterend', DISLIKE_ELE)
+    selector.insertAdjacentHTML('afterend', DISLIKE_ELE);
+    selector.parentElement.addEventListener('click', dislikeHandler);
     manipulate();
 });
 
-let lastUrl = location.href; 
+let lastUrl = location.href;
```
Comment (priyavrat-misra) [inline content.js:65]: FYI I never encountered the below code snippet execute even once.

**Comment by priyavrat-misra** [comment]
FYI I never encountered [this code snippet](https://github.com/bunnykek/LeetCode-Fix/pull/4/files#diff-15c947723c169a1240ecef5c9dba3c267382cbe0ba946365aa6bc2e37f68fe9dR65-R76) execute even once.

**Comment by bunnykek** [comment]
> FYI I never encountered [this code snippet](https://github.com/bunnykek/LeetCode-Fix/blob/c6ae22ac70ac8be644a3b419b43350cbfce34a8c/content.js#L54-L65) execute even once.

Yeah. I also checked after removing it and its working fine. Update your branch with that block removed. Then I will merge this.

**Pull request merged**